static/src/javascripts/projects/common/modules/experiments/tests/frontend-dotcom-rendering-epic.js

```diff
@@ -0,0 +1,205 @@
+// @flow
+
+import { getBodyEnd } from '@guardian/slot-machine-client';
+import { getSync as geolocationGetSync } from 'lib/geolocation';
+import {
+    makeEpicABTest,
+    setupOnView,
+    emitBeginEvent,
+    setupClickHandling,
+    emitInsertEvent,
+} from 'common/modules/commercial/contributions-utilities';
+import reportError from 'lib/report-error';
+import fastdom from 'lib/fastdom-promise';
+import config from 'lib/config';
+import { getMvtValue } from 'common/modules/analytics/mvt-cookie';
+
+const campaignId = 'frontend_dotcom_rendering_epic';
+const geolocation = geolocationGetSync();
+
+const buildKeywordTags = page => {
+    const keywordIds = page.keywordIds.split(',');
+    const keywords = page.keywords.split(',');
+    return keywordIds.map((id, idx) => ({
+        id,
+        type: 'Keyword',
+        title: keywords[idx],
+    }));
+};
+
+const checkResponseOk = response => {
+    if (response.ok) {
+        return response;
+    }
+
+    throw new Error(
+        `Contributions fetch failed with response code: ${response.status}`
+    );
+};
+
+const decodeJson = response => response.json();
+
+const products = ['CONTRIBUTION', 'MEMBERSHIP_SUPPORTER'];
+
+const canRun = (): boolean =>
+    config.get('page').dcrCouldRender &&
+    config.get('tests').dotcomRenderingControl === 'control';
+
+const frontendDotcomRenderingTest = {
+    id: 'FrontendDotcomRenderingEpic',
+    campaignId,
+
+    highPriority: true,
+
+    start: '2020-03-13',
+    expiry: '2020-05-13',
+
+    author: 'Andre Silva',
+    description:
+        'A/B test Default Epic on Frontend vs DCR, both from a remote source, to compare Epic performance.',
+    successMeasure: 'Conversion rate',
+    idealOutcome: 'No difference between control and variant',
+
+    audienceCriteria: 'All',
+    audience: 1, // TBC
+    audienceOffset: 0,
+
+    geolocation,
+
+    canRun,
+
+    variants: [
+        {
+            id: 'frontend', // vs 'dcr' on DCR
```

Comment:
You do need a DCR variant to report I think. It can be noop in here, and then report from DCR.

Reply:
But any variant I add in here will get 50% of the test traffic, no?

Reply:
@tomrf1 @gtrufitt Do you guys expect any issue with a test having only one variant as this one does?

Reply:
Hmm, I'm not sure tbh - I think that's why it's worth putting it out there and seeing what the reporting does, I wasn't sure whether this would report the variant from DCR correctly if the variant is not set up here, but maybe it'll be fine!

Reply:
A test with a single variant is normal in frontend (for us at least!)